fulp_modules/features/antagonists/bloodsuckers/code/vassal/vassal_datum.dm

```diff
@@ -13,7 +13,7 @@
 	hud_icon = 'fulp_modules/features/antagonists/bloodsuckers/icons/bloodsucker_icons.dmi'
 	tip_theme = "spookyconsole"
 	antag_tips = list(
-		"You are a Vassal, enslaved to your vampiric master, you obey their instructions above all else.",
+		"You are a Vassal. Enthralled by your vampiric master, you must (with few exceptions,) obey their instructions above all else.",
```

Comment:
What exceptions are there?
```suggestion
		"You are a Vassal. Enthralled by your vampiric master, you must (with few exceptions), obey their instructions above all else.",
```

Reply:
Instructions which violate server rules.

Reply:
Which? Vassalization afaik is highest priority

Reply:
Upon further consideration I've realized that matters of server policy are beyond me (both in that I have no authority over those matters and I don't want to spend a lot of time thinking about them). I'll remove this incredibly vague parenthetical.